### ISSUE TITLE
Smeltery Controller stuck in Casting Basin

### ISSUE THREAD

**Issue by Evolution0** [SlimeKnights/TinkersConstruct#4319]
**Issue description:**
With the new changes requiring a Smeltery Controller to be made via a Seared Heater cast with Copper I attempted to do so, twice. Both times the resulting controller gets stuck in the casting basin and no amount of right clicking will free it, nor breaking the basin. I also tried outputting with a hopper to no avail.

**If crashed, link to crash report (use a site such as pastebin):**
No crash, no error messages neither server no client side.

**Versions:**
* Minecraft: 1.16.5
* Forge: 36.1.0
* Mantle: 1.6.79
* Tinkers Construct: 3.0.2.78

**Confirm below that this issue is not covered on the roadmap or "Whats New?"**
I checked, its not mentioned that this doesn't work yet.

**Can it be reproduced with *just* Tinkers Construct? If not, list the other mods *required* to reproduce the issue.**
As far as I am aware, however I will do more testing on my own in a more controlled environment. Currently running on an Arclight server so.. who knows, no errors in my log though.

Update: Must be an issue with Arclight, in single player with the same modlist it works perfectly fine, I'll pass this one over to them.

**Comment by KnightMiner** [comment]
One thing to check is if the basin has fluid in it, you could drain the basin into a seared tank below for instance. However, even in that case it should drop when the basin is broken, so server software is a likely culprit. Let me know what ends up being the problem.

**Comment by Evolution0** [comment]
After some testing I've realized its not just the controller getting stuck, anything put into a casting basin or casting table is then not removable, after manually spawning controllers in on my server I setup a full smeltery, dropped some ore in, and attempted to just cast some blocks and ingots, I could not. I also could not drain fluid out of the basin.

The basin and table would act like a black hole, taking in as many items as you clicked on it with. The casting table could not be poured into and the cast isn't removable. 

The casting table would look like this:
![2021-04-07_23 09 05](https://user-images.githubusercontent.com/1984403/113962877-93a8e300-97f6-11eb-9619-fd1f2c9de71a.png)

And if you attempted to pour into a basin again after it does nothing the first time (other than the fade to brown animation on the heater, no sizzle as it cools though) when trying to make a smeltery controller it will do this:
![2021-04-07_23 13 46](https://user-images.githubusercontent.com/1984403/113963222-36616180-97f7-11eb-8060-0be9b00b012e.png)


Otherwise if just trying to make blocks it will allow you to just keep pouring into it endlessly.

**Comment by Evolution0** [comment]
I'll try breaking my setup down to the bare minimum (Forge, Mantle, and Tinkers Construct) on a plugin-less Arclight server and do some more testing soon, might be some additional logging I can enable in configs. Until then I just need to try the dev build of Arclight and wait on some kind of input from the Arclight devs.

Update: I have since done so, no change at all. I did however notice that I get:
```
[14:28:16] [Server thread/WARN] [minecraft/TileEntity]: Skipping BlockEntity with id
```

While placing the Seared Heater on the ground, it still functions normally though.

I've since dug into the logs more and found much more info here:
[debug.log](https://github.com/SlimeKnights/TinkersConstruct/files/6280879/debug.log)

**Comment by Evolution0** [comment]
Fixed in Arclight as of: IzzelAliz/Arclight@1c0c9116d5d3a2788c36a96701b76e3ce2f8ab57
